tests/test_digital.py

```diff
@@ -165,12 +166,12 @@ def test_CSWAP_state110_controlqubit_0() -> None:
         (state_000, state_000),
         (state_001, state_001),
         (state_100, state_100),
-        (state_101, state_110),
-        (state_110, state_101),
+        (state_101, state_011),
+        (state_110, state_110),
     ],
 )
 def test_CSWAP_controlqubits0(initial_state: Tensor, expected_state: Tensor) -> None:
-    cswap = pyq.CSWAP((0, 1), 2)
+    cswap = pyq.CSWAP(2, (0, 1))
```

Comment:
Same as above, best to make the SWAP just `(0, (1, 2))` and fix the tests accordingly. It's a minor thing but just makes it easier to read 😅

Reply:
Modified.